faculty_cli/cli.py

```diff
@@ -1292,3 +1293,99 @@ def ls(project, path):
             click.echo("/project{}/".format(item.path))
         else:
             click.echo("/project{}".format(item.path))
+
+
+@cli.group()
+def dataset():
+    """Manipulate files in Faculty datasets."""
+    pass
+
+
+@dataset.command(name="get")
+@click.argument("project")
+@click.argument("project_path")
+@click.argument("local_path")
+def dataset_get(project, project_path, local_path):
+    """Copy from a project's datasets to the local filesystem."""
+    project_id = _resolve_project(project)
+    faculty.datasets.get(project_path, local_path, project_id=str(project_id))
+
+
+@dataset.command(name="put")
+@click.argument("project")
+@click.argument("local_path")
+@click.argument("project_path")
+def dataset_put(project, local_path, project_path):
+    """Copy from the local filesystem to a project's datasets."""
+    project_id = _resolve_project(project)
+    faculty.datasets.put(local_path, project_path, project_id=str(project_id))
+
+
+@dataset.command()
+@click.argument("project")
+@click.argument("source_path")
+@click.argument("destination_path")
+def mv(project, source_path, destination_path):
+    """Move a file within a project's datasets."""
+    project_id = _resolve_project(project)
+    faculty.datasets.mv(
+        source_path, destination_path, project_id=str(project_id)
+    )
+
+
+@dataset.command()
+@click.argument("project")
+@click.argument("source_path")
+@click.argument("destination_path")
+def cp(project, source_path, destination_path):
+    """Copy a file within a project's datasets."""
+    project_id = _resolve_project(project)
+    faculty.datasets.cp(
+        source_path, destination_path, project_id=str(project_id)
+    )
+
+
+@dataset.command()
+@click.argument("project")
+@click.argument("project_path")
+def rm(project, project_path):
+    """Remove a file from the project's datasets."""
+    project_id = _resolve_project(project)
+    faculty.datasets.rm(project_path, project_id=str(project_id))
+
+
+@dataset.command()
+@click.argument("project")
+@click.argument("project_path")
+def rmdir(project, project_path):
```

Comment:
Similarly to my point on `etag` below, `datasets.rmdir` exists mainly for historical reasons (we did not have recursive `rm` previously so this was the only way to remove an empty directory). In case we decide to deprecate it from `datasets`, can we leave this out for now?